### PR TITLE
Clean the project

### PR DESCRIPTION
**Pull request by rsanath**
* Moved a project that was in the root directory to the /projects folder.
* Removed a discontinued project.
* Removed duplicate code.

### CODE REVIEW

File: scripts/cibuild.rb [hunk 1]
```diff
@@ -95,12 +95,6 @@ def verify_file (f)
       return [f, error]
     end
 
-    if dups.any? then
```
Comment (ritwik12) [inline scripts/cibuild.rb:98]: @ramsanath can you please explain what these changes are?

**Review by ritwik12 — DISMISSED** [review]

File: scripts/cibuild.rb (hunk 1 above)
Comment (rsanath) [reply scripts/cibuild.rb:98]: @ritwik12 I noticed that the code to check for duplicate tags was present twice.
So I removed the unnecessary error check.
Screenshot 2019-08-22 at 1 13 28 PM

File: _data/projects/Please.yml [hunk 2]
```diff
@@ -1,14 +0,0 @@
...
-# Note: these are tags categorizing the project, not issue labels.
-- javascript
-- react
-- web
-- html
-- serverless
-- node
-upforgrabs:
-  name: good first issue
-  link: https://github.com/PleaseDotCom/frontend/labels/good%20first%20issue
```
Comment (shiftkey) [inline _data/projects/Please.yml:14]: @ramsanath I gather this is the project referred to here:

> Removed a discontinued project.

I checked the link here and see there are 3 open issues with this label, which makes me think it's still open for new contributors. Can you elaborate on why you think this project can be removed?

Comment (rsanath) [reply _data/projects/Please.yml:14]: Hi, Please refer to the README of that repository

> Due to an unsuccessful ICO, Please has ceased to exist. Hower the project has been taken over by Deens

Most of the issues there are a year old and there is no source code in that repository.
I had planned to add the new repository 'Deens' in a separate branch.

Comment (shiftkey) [reply _data/projects/Please.yml:14]: Ah, that's the context I was missing! Thanks @ramsanath!

**Comment by rsanath** [comment]
Hi, I believe I have provided the context for the changes made. I'm happy to provide more information to merging this PR 😁.

**Review by shiftkey — APPROVED** [review]
Thanks @ramsanath!

**Comment by rsanath** [comment]
Thanks everyone! 
This was my first open source contribution.

**Pull request merged**